test/trigger_test.sh

```diff
@@ -4,34 +4,35 @@ set -e
 
 # NOTE: this script will execute from the repository root when called on jenkins
 
-VAULT_TOKEN=$1
-INPUTS_JSON=$2
-WDL_FILE=$3
-DEPENDENCIES_JSON=$4
+PIPELINE_FOLDER_NAME=$1
 WD=$(pwd)
 
-echo "Rendering templates"
-bash ${WD}/test/config/render-ctmpls.sh "dev" "${VAULT_TOKEN}"
+
+INPUTS_JSON="/working/test/${PIPELINE_FOLDER_NAME}/pr/test_inputs.json"
```

Comment:
This is good for now. Longer term, I'd **strongly** suggest we make this structure a bit less restrictive; it's annoying (as perhaps you've just discovered!) to realize you need to name all the files _just so_.

Reply:
We can make this a parameter for the test if that is preferred. This file structure / naming convention is used in the portability test so we thought we would just take advantage of the required naming that is already in place to minimize parameters here

Reply:
On one hand, the UX of this is awful, since it's not very discoverable for users trying to write new tests (look at the commit flow for #184 , you can see that I wasn't able to guess right the first time). 

However, I'm also aware that it's used for all the workflow tests, so the scope of fixing this is much larger. Ideally, tests added to the testing folder would be automatically discovered by both the GCP and AWS/Local CI tests. I'd suggest filing away an issue for later?

Reply:
Yes, automatically discovered tests would be much better. I can add a ticket to make that happen in the future